components/ui/button/CustomButton.tsx

```diff
@@ -29,9 +29,9 @@ const CustomButton: React.FC<CustomButtonProps> = ({
   type = ButtonType.PRIMARY,
   size = ButtonSize.BLOCK,
 }) => {
-  function backgroundColor(): string {
-    const disabledType = `${disabled ? Availability.DISABLED : Availability.ENABLED}-${type}`;
+  const disabledType = `${disabled ? Availability.DISABLED : Availability.ENABLED}-${type}`;
```

Comment:
if we changed that to const what will hapen if we changed type in the disabled while using the view for example ?